doc/ref_cert/lfn/chapters/chapter05.md

```diff
@@ -104,40 +107,80 @@ execution and result reporting along with required artifacts and environments in
 place.
 
 #### 5.3.1 Test Case Model
+As there are more number of VNF at different levels of networking such as access,
+transport and core level as well as OSI level L0-L7. Every network functions
+provides set of pre-defined features and functionalities. so its important to
+model test cases for every functionality to identify it uniquely and use it as
+part of test flow design.
+
+As part of modeling its very important to capture the following details
+
+* Test case Name
+* Test case description
+* Virtual Network function Name
+* Network function Feature/functionality name
+* Test case input parameters
+* Test case result attributes
+* Test case version
+
+while implementing the test cases, this model would act as specification and as
+it captures the input and output, it would help while designing the test flow
+which will help to execute set of test cases in pre-defined flow.
 
 #### 5.3.2 Test case management
 
-Test suite
-
-Test flow
+* **Test case** : On-board/discover, update, disable/enable, delete
+* **Test suite** : On-board/discover, update, disable/enable, delete
+* **Test flow** : design/discover, update, disable/enable, delete
 
 #### 5.3.3 Test Execution management
 
-Different run-time support (JVM, Python, Shell, Container, Cloud VM, etc)
+* **Run-time**: One of the common nature of the test environment is heterogeneous
+and multiple vendors and open communities would provide various test tool and
+environment to support execution of test cases developed under different run-times  
+(JVM, Python, Shell, Container, Cloud VM, etc)
+
+* **RPC**: In order to enable the scaling/remote execution, it should be enabled with
+required gRPC or queue support

```

Comment:
We should keep it generic to RPC instead of gRPC. RPC can have multiple implementations - gRPC, via MQ etc.

Reply:
@msidana yeah, but i see now that gRPC and REST becoming an standard protocols in cloud applications and so i tried to mean it with gRPC and queue. but its also good to make it RPC as its very generic. thanks.